e2e/popup-e2e.ts

```diff
@@ -106,7 +112,7 @@ describe('popup', function TestPopup() {
       )
       assert.deepStrictEqual(
         elTexts,
-        ['Stack Overflow', 'Example', 'Links', 'Example', 'Wikipedia'],
+        ['Stack Overflow', 'Example 1', 'Links', 'Example 2', 'Wikipedia'],
```

Comment:
Without the change in sorting algorithm the result would be 
```
[
  "Stack Overflow"
  "Example 2"
  "Links"
  "Example 1"
  "Wikipedia"
]
```
which is invalid